### PR TITLE
Fix print

### PR DESCRIPTION
**Pull request by CalebNeelCO**
Adds all the functionality and fixes that where mentioned in the issues.

### CODE REVIEW

File: templates/tag_summary_template.html [hunk 1]
```diff
@@ -1,222 +1,234 @@
...
+   } else {
+       x.style.display = "none";
+       span.style.display = "inline";
+       buttonimg.src = "https://ljsimpleregisterlookup.herokuapp.com/static/images/details_open.png"
+   
+   }
+   }
 </script>
- {% endif %}
+{% endif %}
```
Comment (rolsen) [inline templates/tag_summary_template.html]: Since the indentation has been changed here, it's hard to see what actually changed in the code. I'm just going to call it fine. This means I'm not reviewing this section of code. Please take care in the future to not mess with existing indentation so that diffs are minimal.

tldr; I'm assuming you code is flawless!

**Comment by rolsen** [comment]
Merging this will close #8.

File: templates/tag_summary_template.html (hunk 1 above)
Comment (CalebNeelCO) [reply templates/tag_summary_template.html]: I re indented the whole file, it was pretty bad before I started, I have a version that only has my changes. Would you like me to use this one?

File: templates/scribe_prefix.html [hunk 2]
```diff
@@ -111,4 +111,43 @@
...
+     /*
+     use for fade effect insted of elipse on tag preview.
+    background: -webkit-linear-gradient(left, #000000 , #777676);
+   background: -o-linear-gradient(right, #000000, #777676);
+   background: -moz-linear-gradient(right, #000000, #777676);
+   background: linear-gradient(to right, #000000 , #777676); 
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  */
+    }
```
[the remark this answers is not in the document]
Comment (CalebNeelCO) [reply templates/scribe_prefix.html]: Done

**Comment by rolsen** [comment]
Thanks for your patience!
